src/main/java/ch/njol/skript/expressions/ExprEnchantmentOffer.java

```diff
@@ -0,0 +1,96 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
+ */
+package ch.njol.skript.expressions;
+
+import org.bukkit.enchantments.EnchantmentOffer;
+import org.bukkit.event.Event;
+import org.bukkit.event.enchantment.PrepareItemEnchantEvent;
+import org.eclipse.jdt.annotation.Nullable;
+
+import ch.njol.skript.ScriptLoader;
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Events;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.skript.log.ErrorQuality;
+import ch.njol.util.Kleenean;
+
+@Name("Enchantment Offer")
+@Description("The enchantment offer in enchant prepare events.")
+@Examples({"on enchant prepare:",
+		"\tsend \"Your enchantment offers are: %the enchantment offers%\" to player"})
+@Since("INSERT VERSION")
+@Events("enchant prepare")
+@RequiredPlugins("1.11 or newer")
+public class ExprEnchantmentOffer extends SimpleExpression<EnchantmentOffer> {
+
+	static {
+		if (Skript.classExists("org.bukkit.enchantments.EnchantmentOffer")) {
+			Skript.registerExpression(ExprEnchantmentOffer.class, EnchantmentOffer.class, ExpressionType.SIMPLE, 
+					"enchant[ment] offer 1",
```

Comment:
in my opinion, this expression should not support that singular syntax - you should simply have `enchantment offers` then another expression can handle things like `first element out of enchantment offer`

Reply:
I think it'd be nice to be able to write

`enchantment level of enchantment offer 1`
than
`enchantment level of first element out of enchantment offers`

Reply:
alright, but to be consistent with other expressions the pattern should be `1st enchantment offer` and the number should be an expression otherwise you can't do `enchantment offer {_something}`

https://github.com/SkriptLang/Skript/blob/4c37bc93768db3d0d303c865ca4fd20d29976d44/src/main/java/ch/njol/skript/expressions/ExprLore.java#L70